docs/pages/inboxes/user-consent/support-user-consent.md

```diff
@@ -146,13 +120,9 @@ try await conversation.updateConsent(.allowed) // .allowed | .denied
 
 :::
 
-## Stream consent records in real-time
-
-Listen for real-time updates to consent records:
-
-:::tip[Note]
-Consent syncing is backed by XMTP's history system. Syncing works only if a [history sync URL](/inboxes/create-a-client/#configure-an-xmtp-client) is specified on client create. By default, the URL to an Ephemera-hosted history sync server is set.
-:::
+## Stream consent preferences in real-time 
```

Comment:
Just making sure - does `streamAll` also stream consent preference updates and new HMAC keys?

https://docs.xmtp.org/inboxes/list-and-stream#stream-all-group-chat-and-dm-messages

Reply:
yes streamAll should stream consent and preference updates now. cc @codabrink to triple confirm.